src/Util.ts

```diff
@@ -35,14 +36,16 @@ const agentSelector = function (_parsedURL: any) {
  * Request the provided API endpoint with the provided POST data. All requests to the TSP today are in POST. On failure,
  * attempt to parse the failed JSON to extract an error code and message.
  */
-export const makeJsonRequest = <T>(tspDomain: string, apiKey: string, route: string, postData: string): Future<TenantSecurityException, T> =>
+export const makeJsonRequest = <T,>(tspDomain: string, apiKey: string, route: string, postData: string): Future<TenantSecurityException, T> =>
     Future.tryP(() =>
         fetch(`${tspDomain}/api/1/${route}`, {
             method: "POST",
             body: postData,
             headers: {
                 "Content-Type": "application/json",
                 Authorization: `cmk ${apiKey}`,
+                "x-icl-tsc-language": "nodejs",
```

Comment:
The TSP currently accepts "node" but not "nodejs", so we'll have to change one of the two to make them line up

Reply:
Interesting that it accepts specific headers for that instead of forwarding any string... it seems like we don't want to have to update the TSP anytime we add another TSC.

Reply:
Chris and I discussed for a while and we decided that limiting languages to protect against someone exploding the cardinality was worth it to a degree. So we currently have very limited languages allowed, and we have some validation on the version to keep that safe-ish.

Reply:
Huh. That doesn't seem worth it to me. Someone needs valid access to a TSP to do that, and that'd be a very strange thing to do with no significant benefit, that wouldn't be crazy hard to fix if it did happen. Adding more TSCs is something that is definitely going to happen multiple times. Guess it's done though, I don't think I care enough to force it to change.